doc/specs/vulkan/registry.txt

```diff
@@ -814,6 +818,10 @@ members.
     contains a `latexmath:` equation, this attribute should
     be specified with an equivalent equation using only C builtin operators,
     C math library function names, and variables as allowed for attr:len.
+    It must be fully valid C99 expression having equivalent result to
```

Comment:
"must be a valid C99 expression whose result is equal to attr:len for all possible inputs" in both places.